crates/sargon/src/signing/testing/simulated_user.rs

```diff
@@ -123,8 +121,8 @@ impl SimulatedUser {
     }
 }
 
-unsafe impl Sync for SimulatedUser {}
-unsafe impl Send for SimulatedUser {}
+unsafe impl<S: Signable> Sync for SimulatedUser<S> {}
+unsafe impl<S: Signable> Send for SimulatedUser<S> {}
```

Comment:
same here as in https://github.com/radixdlt/sargon/pull/288/files#r1865977941

Reply:
Does not seem to work